STYLE_GUIDE.md

```diff
@@ -0,0 +1,75 @@
+# Introduction
+
+This is the code style guide for the FAST tooling repository. When contributing to this project please follow the standards defined in this guide.
+
+## Files
+
+File and folder naming in general should not include special characters or space. Alphanumeric is preffered, separated by dash or underscore depending on the type of file. Names should be as concise as possible while also being meaningful.
+
+### Organization
+
+Folder structure should be as flat as possible. This is not to say that all files should go into a single folder, folders should exist where they make sense for larger categorizing purposes and file names can be leveraged to supply additional context.
+
+Example:
+```text
+my-component/
+└─ my-component.ts
+└─ my-component.style.ts
+└─ my-component.spec.ts
```

Comment:
In the fast-components `.styles` (plural) is used. Might want to also add `.template` as well for an example of how templates are sometimes broken out.